lib/utils/aws/aws.go

```diff
@@ -227,22 +236,23 @@ func VerifyAWSSignature(req *http.Request, credProvider aws.CredentialsProvider)
 }
 
 // NewSignerV2 is a temporary AWS SDK migration helper.
-func NewSignerV2(provider aws.CredentialsProvider, signingServiceName string) *v4.Signer {
-	return NewSigner(migration.NewCredentialsAdapter(provider), signingServiceName)
-}
-
-// NewSigner creates a new V4 signer.
-func NewSigner(credentials *credentials.Credentials, signingServiceName string) *v4.Signer {
-	options := func(s *v4.Signer) {
+func NewSignerV2(signingServiceName string) *v4.Signer {
```

Comment:
are we done with the migration for signer migration? if so i would rename this to just `NewSigner` and update the function description